Changes

```diff
@@ -5,6 +5,9 @@ Working version
 
 ### Language features:
 
+- MPR#6023, GPR#1648: Allow type-based selection of GADT constructors
+  (Thomas Refis, review by Jacques Garrigue and Gabriel Scherer)
```

Comment:
I think Leo could be in here (as a reviewer?) as he did some nice explanation work that was useful for at least my own understanding of your work.

Reply:
Indeed, I've fixed this.